### PR TITLE
ODIM-3545 Code implementation of get on connection method

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

**Review by bharath-b-hpe — APPROVED** [review]

**Review by skhan-anjum — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by a-ajith — APPROVED** [review]

File: svc-aggregation/agcommon/common.go [hunk 1]
```diff
@@ -22,8 +22,19 @@ import (
 	"github.com/ODIM-Project/ODIM/lib-utilities/common"
 	"github.com/ODIM-Project/ODIM/lib-utilities/config"
 	"github.com/ODIM-Project/ODIM/svc-aggregation/agmodel"
+	uuid "github.com/satori/go.uuid"
 )
 
+// SupportedConnectionMethodTypes is for validating the connection method type
+var SupportedConnectionMethodTypes = map[string]bool{
+	"Redfish": true,
+	"SNMP":    true,
```
Comment (Bharath-KKB) [inline svc-aggregation/agcommon/common.go]: set to false for everything other than redfish

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
change as per comment

File: svc-aggregation/agcommon/common.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-aggregation/agcommon/common.go]: done

Comment (irappa-pattar) [reply svc-aggregation/agcommon/common.go]: done

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**